common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -334,11 +334,7 @@ def setUp(self, *args, **kwargs):
         super(CheckboxProblemTypeTest, self).setUp(*args, **kwargs)
         self.problem_page.a11y_audit.config.set_rules({
             'ignore': [
-                'section',  # TODO: AC-491
-                'aria-allowed-attr',  # TODO: AC-251
-                'aria-valid-attr',  # TODO: AC-251
-                'aria-roles',  # TODO: AC-251
-                'checkboxgroup',  # TODO: AC-251
+                'section'  # TODO: AC-491
             ]
         })
 
```

Comment:
Removing these rules from ignore for just spike. i.e checkboxes and radiogroups.